data/examples/simba.cfg

```diff
@@ -63,6 +63,11 @@ extended_output_plots = false
 rotation_filter_variable = null
 # Write a new trips.csv during report mode to output directory? (default: false)
 create_trips_in_report = false
+# Pickle current schedule and scenario during report mode
+# create_pickle_in_report = false
+# Load this pickle file, expects load_pickle as first mode
+# load_pickle = "example.pkl"
```

Comment:
make verbose that its a path. for me "load_pickle" reads like a bool

Reply:
Changed to load_pickle_path